spec/mongoid/traversable_spec.rb

```diff
@@ -251,4 +251,164 @@
       end
     end
   end
+
+  describe "#discriminator_key" do
+
+    context "when the discriminator key is not set on a class" do
+      it "equals _type" do
```

Comment:
Could you make this test description more descriptive?

Reply:
done!